<?xml version="1.0" encoding="UTF-8"?> 
  <!-- 
  navx.launch
  VERSION: 1.0.0 
  LAST_CHANGED: 2019-10-31
  AUTHOR: Nick Schatz
  -->
<launch>
  <node pkg="navx" type="navx_node" name="navx" output="screen" respawn="true">
    <param name="frequency" value="50.0"/>
    <param name="publish_euler" value="false"/>
    <param name="device_path" value="/dev/ttyACM0"/>
    <param name="frame_id" value="imu_link"/>
    <param name="covar_samples" value="100"/>
  </node>
</launch>